src/LegacyRepContract.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/token/VariableSupplyToken.sol';
+import 'ROOT/libraries/token/ERC20.sol';
```

Comment:
Recommend removing this import, its not directly used anywhere in this contract.